.pre-commit-config.yaml

```diff
@@ -70,7 +70,8 @@ repos:
     exclude: |
         (?x)^(
             singer_sdk/helpers/_simpleeval.py|
-            tests/core/test_simpleeval.py
+            tests/core/test_simpleeval.py|
+            singer_sdk/testing/runners.py
```

Comment:
I was having issues whereby the `pyupgrade` changes were failing our lint rules 🤦‍♂️ It was easier at the time to just ignore the file, but will dig into it a bit now.

Reply:
Failure [here](https://github.com/meltano/sdk/actions/runs/3693498459/jobs/6253558012) and mypy issue [here](https://github.com/python/mypy/issues/8897). Seems like its either best to ignore this from `pyupgrade` or ignore it from `mypy`. I went for the former, so we can still benefit from `mypy` checks 🙂

Reply:
@kgpayne it's because you have `type` as a class attribute and it's shadowing the built-in object:

```python
class TapTestRunner(SingerTestRunner):
    """Utility class to simplify tap testing."""

    type = "tap"
```

You should probably change that to something like `plugin_type`.